mergify_engine/actions/queue.py

```diff
@@ -306,7 +306,11 @@ async def _should_be_merged(
         if not await q.is_first_pull(ctxt):
             return False
 
-        if not await ctxt.is_behind:
+        if await ctxt.is_behind:
```

Comment:
I guess this is cached, no? Are you sure it's up to date when checking here?

Reply:
Yes it's up to date.